api/api.go

```diff
@@ -102,6 +102,7 @@ const (
 	apiV2AlertRuleFromGUID = "v2/AlertRules/%s"
 
 	apiV2CloudAccounts        = "v2/CloudAccounts"
+	apiV2CloudAccountsByType  = "v2/CloudAccounts/%s"
 	apiV2CloudAccountFromGUID = "v2/CloudAccounts/%s"
```

Comment:
Wonder should there be a distinction between these 2 consts. They're both the same. Thoughts on combining them in to 1 const? `apiV2CloudAccountsWithParam` or something to that effect.

Reply:
I think that's reasonable. Let me update